components/automate-ui/src/app/page-components/profile/profile.component.html

```diff
@@ -27,8 +27,14 @@
         </li>
       </app-authorized>
       <li class="dropdown-list-item">
-        <span class="version">Version: 2</span>
-        <span class="build">Build: {{ buildVersion }}</span>
+        <ul class="dropdown-sub-list">
```

Comment:
It doesn't appear that dropdown-sub-list does anything specific but I opted to include it for consistency with the rest of the lists in the menu.